src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudServiceTest.kt

```diff
@@ -132,40 +138,87 @@ class CosmasGoogleCloudServiceTest {
                 any(BlobInfo::class.java), any(ByteArray::class.java)))
                 .thenThrow(StorageException(1, "test"))
         this.service = CosmasGoogleCloudService(this.BUCKET_NAME, fakeStorage)
-        val createVersionRecorder: StreamRecorder<CosmasProto.CreateVersionResponse> = StreamRecorder.create()
-        val newVersionRequest = CosmasProto.CreateVersionRequest
-                .newBuilder()
-                .setFileId("0")
-                .setProjectId("0")
-                .setFile(ByteString.copyFromUtf8("text"))
-                .build()
-        this.service.createVersion(newVersionRequest, createVersionRecorder)
-        assertNotNull(createVersionRecorder.error)
-        assertEquals("test", createVersionRecorder.error!!.message)
-        this.service = getServiceForTests()
+        createVersion("file")
+        val (commitRecorder, commitRequest) = getStreamRecorderAndRequestForCommitVersions()
+        this.service.commitVersion(commitRequest, commitRecorder)
+        assertNotNull(commitRecorder.error)
+        assertEquals("test", commitRecorder.error!!.message)
+    }
+
+    @Test
+    fun addTwoVersionsAndOneCommit() {
+        createVersion("ver0")
+        createVersion("ver1")
+        commit()
+        assertEquals("ver1", getFileFromService(0))
+    }
+
+    @Test
+    fun makeTwoCommits() {
+        val fakeStorage: Storage = mock(Storage::class.java)
+        val blob1 = getMockedBlob("ver2", 0)
```

Comment:
In `getMockedBlob` the first argument name is misleading. It it file content, not file as it says.

Reply:
done